Description: Fix requirements.txt
 The current requirements.txt is producing a non-working dependency list.
 Just get the broken lines out.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2016-06-07

--- keystone-10.0.0~b1.orig/requirements.txt
+++ keystone-10.0.0~b1/requirements.txt
@@ -10,9 +10,6 @@ pbr>=1.6 # Apache-2.0
 WebOb>=1.2.3 # MIT
 PasteDeploy>=1.5.0 # MIT
 Paste # MIT
-Routes!=2.0,!=2.1,!=2.3.0,>=1.12.3;python_version=='2.7' # MIT
-Routes!=2.0,!=2.3.0,>=1.12.3;python_version!='2.7' # MIT
-cryptography!=1.3.0,>=1.0 # BSD/Apache-2.0
 six>=1.9.0 # MIT
 SQLAlchemy<1.1.0,>=1.0.10 # MIT
 sqlalchemy-migrate>=0.9.6 # Apache-2.0
